ios/MaasCore/Sources/MaasTheme/Theme/Spacing.swift

```diff
@@ -29,4 +26,16 @@ public struct Spacing {
     public static var `default`: Spacing {
         .init(value: CGFloat(SpacingScale.Default().GlobalMargin))
     }
+
+    public static var listItemHorizontal: Spacing {
```

Comment:
This is a very separate topic, but we need to agree with Algis whether we want to expose `listItemHorizontal`, `listItemVertical` and `listItemGutter` as globally configurable Bedrock theme properties for our SDK users. If so, these will need to be added to the Android implementation, too.

Reply:
If these are unused in this PR now, can remove them and we'll address it later

Reply:
These are used in Cell.swift aka (ListItem). These few constants appear in Bedrock weeks when develop nearby transit ui components. 😭